src/js/components/CheckBox.js

```diff
@@ -6,13 +6,34 @@ const CLASS_ROOT = "check-box";
 
 export default class CheckBox extends Component {
 
+  set checked (value) {
+    if ('refs' in this) {
+      this.refs.input.checked = !!value;
+    }
+  }
+
+  get checked () {
+    return 'refs' in this ? this.refs.input.checked : null;
+  }
+
   render () {
-    var classes = [CLASS_ROOT];
-    var labelId = 'checkbox-label';
-    var hidden;
+    let classes = [CLASS_ROOT];
+    let labelId = 'checkbox-label';
+    let label;
+    let hidden;
+
+    if (this.props.label) {
+      label = (
+        <span role="label" id={labelId} tabIndex="-1" className={CLASS_ROOT + "__label"}>
```

Comment:
can you add `key="label"` to this span to avoid react warnings for unique components inside an array?